setup.py

```diff
@@ -35,6 +35,7 @@ def read(fname):
           'Programming Language :: Python :: 2.7',
           'Programming Language :: Python :: 3',
           'Programming Language :: Python :: 3.2',
-          'Programming Language :: Python :: 3.3'
+          'Programming Language :: Python :: 3.3',
+          'Development Status :: 7 - Inactive'
       ],
       )
```

Comment:
I don't think this works -- `pip` doesn't run `setup.py install` with a `wheel` distribution. And if we don't upload a `wheel`, it looks like, from my testing, that `pip` suppresses the output.